src/imitation/algorithms/preference_comparisons.py

```diff
@@ -24,6 +24,7 @@
 import torch as th
 from scipy import special
 from stable_baselines3.common import base_class, type_aliases, vec_env
+from stable_baselines3.common.utils import check_for_correct_spaces
```

Comment:
Violates section 2.2 of style guide https://google.github.io/styleguide/pyguide.html#22-imports
> Use import statements for packages and modules only, not for individual classes or functions.